functionaltests/src/test/java/specs/scheduler/ScheduledJobFixtureHelper.java

```diff
@@ -58,26 +63,32 @@ public MultiValueResult createPayment(String paymentName,
 
     /**
      * Verify the {@code paymentName} is automatically handled by (short) scheduled task.
+     *
      * @param paymentName payment alias
      * @return map of the payment properties after waiting scheduled task.
      * @throws Exception any exceptions in the tests
      */
     public MultiValueResult verifyPaymentIsHandled(String paymentName) throws Exception {
         Payment payment = paymentFixture.fetchPaymentByLegibleName(paymentName);
 
+        logger.info("Current payment [{}] create at {}, now is {}",
```

Comment:
So a log message could be as folllows: `"Current payment [123] create at 12.01.2018 10:10:10, now is 01.03.2018 08:00:00" `

I see several problems with such message:
1. Why "Current"? doesn't make sense when seen in the logs, I guess.
2. `"Current payment [123]"`, when I see that, I don't know what is "123", is it key? is it id? is it name?.
3. The sentence has wrong grammar, which could lead to a different meaning: payment created at x, is now created at y. (even if it doesn't make sense, but still having ", now is x" conveys that something changed from x -> y.

So I would recommend reformatting it to:
`"Payment with id:[123] is created at 12.01.2018 10:10:10. The current time now is 01.03.2018 08:00:00" `

Reply:
5da23f16